java-frontend/src/test/java/org/sonar/java/bytecode/se/testdata/ExceptionEnqueue.java

```diff
@@ -78,6 +78,37 @@ static boolean testCatchBlockEnqueue2() throws IOException {
     throw new RuntimeException();
   }
 
+  static boolean enqueueExitBlock() throws IOException {
+    throwSpecificException();
+    return false;
+  }
+
+  static boolean enqueueExitBlock2(ExceptionEnqueue ee) throws IOException {
+    try {
+      ee.throwIOException();
+    } catch (FileNotFoundException e) {
+      return true;
+    }
+    return false;
+  }
+
+  static boolean enqueueExitBlock3() throws IOException {
+    try {
+      throwSpecificException();
+    } finally {
+      int x = 0;
+    }
+    return false;
+  }
+
+  static boolean enqueueExitBlock4() {
+    try {
+      throwSpecificException();
+    } catch (Throwable e) {
+      return true;
```

Comment:
wonder if it would not be more interesting to have `false` here to see that we are correctly jumping into the catch.

Reply:
indeed. I modified the unreachable return below to return `false`, so test remains the same, but it is more explicit.